saleor/graphql/account/types.py

```diff
@@ -271,6 +271,13 @@ class User(ModelObjectType):
             description="Slug of a channel for which the data should be returned."
         ),
     )
+    checkouts = ConnectionField(
+        CheckoutCountableConnection,
+        description="Returns checkouts assigned to this user.",
```

Comment:
We need to mention version in which it was added, see usages of ADDED_IN_38 and here as well.